Apps/phcloudflare/readme.html

```diff
@@ -0,0 +1,19 @@
+<!-- File: readme.html
+  Copyright (c) 2021 Splunk Inc.
+
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+
+<html>
+  <head></head>
+  <body>
+    <h2 id="api-token">API Token</h2>
+    <p>API Tokens provide a way to authenticate with the Cloudflare API and can be generated from the <a href="https://dash.cloudflare.com/profile/api-tokens"><span class="link">User Profile &#39;API Tokens&#39; page</span></a>.
+       To support the available actions, please assign the following permissions to your token:</p>
+    <ul>
+      <li><code>Zone.Zone Settings</code>,</li>
+      <li><code>Zone.Zone</code>,</li>
+      <li><code>Zone.Firewall Services</code></li>
+    </ul>
+  </body>
```

Comment:
Please add port details. Matched regex 'Cloudflare' (1 time) using app's name/product_name JSON fields in the local copy of IANA common service ports file. (See https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml) Please check if this app requires specific ports open on the Phantom instance and add them to the app's documentation.

Reply:
The app is only doing HTTPS outgoing calls to Cloudflare API as specified [here](https://api.cloudflare.com/#getting-started-endpoints), no other open ports are required. Does this clarify your request? If not, could you please provide an example? Thanks.

Reply:
If no requirements for any additional ports, then, it's okay with me. Thanks for the quick response!!